hooks/PNKEP.js

```diff
@@ -62,33 +62,25 @@ Field Number:
 4: Checksum
 */
 
-function isEmpty(mixed) {
-  return (
-    (typeof mixed !== "string" && typeof mixed !== "number") ||
-    (typeof mixed === "string" && mixed.trim() === "")
-  )
-}
-
 module.exports = function(parser, input) {
   try {
-    const { id, sentence, parts, tags } = input
-
-    var delta
+    const { id, sentence, parts, tags } = input;
```

Comment:
A lot of the changes in this file are about adding ';' to end of lines. I did not do that in my other PR because it looked like that was not the style chosen by the original author. 

Is there a SignalK "coding guidelines" document somewhere?